test/team_collection_test.rb

```diff
@@ -0,0 +1,22 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/team_collection'
+
+class TeamCollectionTest < Minitest::Test
+  def setup
+    @team_collection = TeamCollection.new('./data/teams_fixture.csv')
+  end
+
+  def test_it_exists
+    assert_instance_of TeamCollection, @team_collection
+  end
+
+  def test_it_can_return_array_of_all_teams
+    @team_collection.create_teams
+    
+    assert_instance_of Array, @team_collection.teams
+    assert_equal 20, @team_collection.teams.size
+
+    assert_equal true, @team_collection.teams.all? { |team| team.class == Team }
```

Comment:
Cool way to check the array of all team objects with that enumerable!

@sarynm12 I really like the way Leah tested this collection here. Maybe you can refactor your game collection test with something similar?